docs/recipes/authentication.md

```diff
@@ -39,9 +39,9 @@ export class AuthState {
 
   @Action(Login)
   login({ patchState }: StateContext<AuthStateModel>, { payload }: Login) {
-    return this.authService.login(payload).pipe(tap((result: { token: string }) => {
+    return this.authService.login(payload).pipe(tap(({ token }: { token: string }) => {
```

Comment:
Could you change back to `result: { token: string }`? We prefer using the simple syntax in the docs :)